packages/block-editor/src/components/gradients/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { find } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+export function __experimentalGetGradientClass( gradientSlug ) {
```

Comment:
Any reason for this to be experimental? I'm thinking we're probably adding too much experimental APIs, especially since 5.4 is too far ahead.

Reply:
The reason for this to experimental is that it is not clear yet if we can merge this API with the color ones. I guess this decision will also depend on the design direction.